### ISSUE TITLE
Remove use of types module constants in twisted/persisted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8454]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8454|
|Type|enhancement|
|Created|2016-06-12 04:37:14Z|
|Branch|https://github.com/twisted/twisted/tree/8454-rodrigc-types|

The *types* module constants have been removed in Python 3.
For example using *types.TupleType* in Python 3 will give an *AttributeError*

These constants should be removed and replaced with the primitive type instead.  See:

http://www.diveintopython3.net/porting-code-to-python-3-with-2to3.html#types

<details><summary>Searchable metadata</summary>

```
trac-id__8454 8454
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8454_rodrigc_types 8454-rodrigc-types
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1465706234313190 1465706234313190
changetime__1466097656909542 1466097656909542
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/174

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Changes look good.

Waiting for test result to check the coverage and will merge if all is ok.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

Looks like the line are somehow covered :) Will merge

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 833c7a1d8bea77b295355dccaccacd6210c3e95a
```
#!CommitTicketReference repository="" revision="833c7a1d8bea77b295355dccaccacd6210c3e95a"
Merge 8454-rodrigc-types: Remove use of types module constants in twisted/persisted.

Author: rodrigc
Reviewer: adiroiban
Fixes: #8454
```
